gepetuto/main.py

```diff
@@ -137,21 +143,32 @@ def get_files(args):
     return files
 
 
+def show_version():
+    """Print gepetuto version."""
+    __metadata__ = metadata("gepetuto")
+    __version__ = __metadata__["version"]
+    print("gepetuto ", __version__)
+
+
 def main():
     """Run command."""
     args = parse_args()
-    if args.directory:
-        os.chdir(args.directory)
-        args = parse_args()
-    files = get_files(args)
-    if args.action == "generate":
-        generate(**vars(args))
-    elif args.action == "lint":
-        lint(files, **vars(args))
-    elif args.action == "test":
-        test(files, **vars(args))
-    elif args.action == "all":
-        LOG.debug("no action specified, running all 3.")
-        lint(files, **vars(args))
-        test(files, **vars(args))
-        generate(**vars(args))
+    if args.version:
+        show_version()
```

Comment:
```suggestion
    if args.version:
        show_version()
        return
```
To avoid an extra indent everywhere else